app/controllers/mechanics_controller.rb

```diff
@@ -0,0 +1,14 @@
+class MechanicsController < ApplicationController
+  def show
+    @mechanic = Mechanic.find(params[:id])
+  end
+
+  def edit
```

Comment:
a more RESTful approach here would be to use a joins controller. Consider having the form in your show page direct to a `create` action in a RideMechanicsController. This is more RESTful/conventional because what we're doing here is creating a joins record for an existing ride and mechanic, rather than updating an attribute of a mechanic. Also,  we usually reserve the `edit` action to render a form to update a resource.